Devscord.DiscordFramework/Commons/Exceptions/InvalidArgumentsException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Devscord.DiscordFramework.Commons.Exceptions
+{
+    public class InvalidArgumentsException : Exception
+    {
+        public string AvailableArguments { get; }
+
+        public InvalidArgumentsException(string arguments)
+        {
+            AvailableArguments = arguments;
```

Comment:
```suggestion
            this.AvailableArguments = arguments;
```

Reply:
To robi jakoś różnice w ogóle?

Reply:
wizualną

Reply:
No właśnie widzę, że to tylko dekoracja XD